Index: javascript/landing/carousel.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- javascript/landing/carousel.js	(date 1604929153000)
+++ javascript/landing/carousel.js	(date 1604929153000)
@@ -19,7 +19,6 @@
             carouselContainer.appendChild(carouselDiv);
 
             carouselDiv=1;
-
         }
         if(next===false) {
             currentPic--;
@@ -42,10 +41,6 @@
     }
 
     carousel[carouselDiv].style.backgroundImage="url('"+immageArray[currentPic]+"')";
-
-    if(carouselDiv.length===1){
-        carousel[0].remove();
-    }
 }
 arrowL.addEventListener('click',function() {
     console.log('test');
